Source/PeleC.H

```diff
@@ -543,6 +610,10 @@ public:
     trans_parms;
   static pele::physics::turbinflow::TurbInflow turb_inflow;
 
+#ifdef PELEC_USE_SOOT
+  static SootModel* soot_model;
```

Comment:
I don't think this should be a pointer? If it needs to be a pointer, can you please replace with a `unique_ptr`?

Reply:
Good point. It probably doesn't need to be a pointer.

Reply:
thanks!